api/XMLHttpRequest.json

```diff
@@ -985,9 +985,15 @@
                 "version_removed": "15"
               }
             ],
-            "opera_android": {
-              "version_added": "42"
-            },
+            "opera_android": [
+              {
+                "version_added": "42"
```

Comment:
This doesn't seem right, it doesn't match Opera 18 above. Some mirroring has gone wrong here.

Reply:
Opera Android was originally set to "42" for this feature so I stuck with that.  However, I realize that doesn't really make sense for a feature like this one, so I'm replacing "42" with "18".